README.md

```diff
@@ -6,10 +6,32 @@ Supported version of Geometry Dash: 1.0 - 2.11 (so any version of Geometry Dash
 
 Required version of PHP: 5.4+ (tested up to 7.1.3)
 
-### Setup
+### Setup for Webhostings
 1) Upload the files on a webserver
 2) Import database.sql into a MySQL/MariaDB database
-3) Edit the links in GeometryDash.exe (some are base64 encoded since 2.1, remember that)
+3) Edit the connection info at /config/connection.php(Set server to localhost if you're using localhost database)
```

Comment:
Don't you mean that when using localhost database with connection.php? Because _when_ you are about to publish the modified gd version, the players then tries to connect to their databases on their pc, which they probably don't have.

Reply:
This is webhost config which is only runable on server itself.